extensions/big-data-cluster/src/mainController.ts

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import * as vscode from 'vscode';
+import { CreateClusterWizard } from './wizards/create-cluster/createClusterWizard';
+
+/**
+ * The main controller class that initializes the extension
+ */
+export class MainController {
```

Comment:
IIRC, this is generated code, for the code we have for now, it seems unnecessary, but I will keept it this way, in case we need to test the controller separately in the future as it gets more complicated.